### PR TITLE
Update contributing link in README

### PR DESCRIPTION
**Pull request by Moini**

### CODE REVIEW

**Comment by Moini** [comment]
(going to update Kielux info a bit more, now that I have seen that there are some more options for it)

**Comment by weeman1337** [comment]
Thank you for your contribution :)

If you have any questions feel free to ask.

**Comment by Moini** [comment]
Danke!

**Comment by weeman1337** [comment]
I deployed it to  https://test.foss.events/ . @dreirik if you're fine with it please merge it.

**Comment by dreirik** [comment]
thank you @Moini for this PR and the excellent review of our about page. Merging it now.

File: src/templates/about.html [hunk 1]
```diff
@@ -264,9 +266,13 @@ <h2>about: contributing</h2>
         can <a
             href="mailto:onemore@foss.events?subject=one more event&body=Hi all,%0D%0A%0D%0Aplease add/update this event on // foss.events:%0D%0A%0D%0AName:%0D%0ADate:%0D%0AURL:%0D%0AShortname:%0D%0AHashtag:%0D%0ACity:%0D%0ACountry:%0D%0AVenue:%0D%0AOSM-Link:%0D%0ASelf-Description:%0D%0ALink for Call for Participation:%0D%0ADeadline for Call for Participation:%0D%0ALink for Code of Conduct:%0D%0AEntrance Fee:%0D%0ARegistration required:%0D%0AParticipants last time:%0D%0AMain language:%0D%0AAny specialties: ">use
         this advanced email-template</a> and help us with collecting all necessary information about an event. If you
-        have trouble understand our data-set, check the legend <a href="#legend">[&uarr;]</a> for an explanation.
+        have trouble understanding our data set, check the legend <a href="#legend">[&uarr;]</a> for an explanation.
     </p>
 
+    <p>
+        Or, if feel comfortable to send us the information about your event as a pull request, you can also edit the tab-separated data file for the year of your event on <a href="https://github.com/foss-events/website/tree/master/data">GitHub</a>.
+    </pA
```
Comment (Moini) [inline src/templates/about.html]: Oops. That end tag doesn't look right... Sorry, will fix.

**Pull request merged**